### PR TITLE
drumcircle: new recipe

### PR DESCRIPTION
**Pull request by davidkaroly**

### CODE REVIEW

**Comment by davidkaroly** [comment]
I'd like some help with this one: DrumCircle expects profiles to use mime type `application/x-vnd.SY.DrumCircle-Profile`
This seems to be included in the hpkg file correctly but when I activate the package, the file system somehow forgets these extra attributes.

**Comment by humdingerb** [comment]
Works here. All profile files in /boot/system/settings/drumcircle/ have application/x-vnd.SY.DrumCircle-Profile as BEOS:TYPE and the app seems to work.

**Comment by davidkaroly** [comment]
Probably it was a build environment issue on my side. After rebuilding the package from 1.0.4 tag, I get the correct MIME types.

File: media-sound/drumcircle/drumcircle-1.0.4.recipe [hunk 1]
```diff
@@ -0,0 +1,41 @@
+SUMMARY="Simple sequencer with Hippie AI"
+DESCRIPTION="A simple MIDI sequencer with a knack for simulating Hippie Drum Circles. \
+Tweek the Tweekables and have fun!"
+HOMEPAGE="https://github.com/steffenyount"
```
Comment (augiedoggie) [inline media-sound/drumcircle/drumcircle-1.0.4.recipe:4]: https://github.com/steffenyount/DrumCircle  would be better

**Comment by augiedoggie** [comment]
I just noticed that this recipe was put in `media-sound` instead of `haiku-apps`.  I kind of like having the native apps all in one place under `haiku-apps` but I don't feel strongly about it.  Should this be moved or should we stop using `haiku-apps`?

**Pull request merged**